plugins/grafana-custom-plugins/grafana-http-table-plugin/package.json

```diff
@@ -0,0 +1,76 @@
+{
+  "name": "grafana-table-plugin",
+  "version": "1.0.0",
+  "description": "",
+  "scripts": {
+    "build": "webpack -c ./.config/webpack/webpack.config.ts --env production",
+    "dev": "webpack -w -c ./.config/webpack/webpack.config.ts --env development",
+    "test": "jest --watch --onlyChanged",
```

Comment:
Remove watch, otherwise it won't stop when we do the typescript unit test in the workflow.